module/library/categories/helper.js

```diff
@@ -32,10 +32,13 @@ module.exports = class LibraryCategoriesHelper {
       * @returns {Object} List of library projects.
      */
 
-    static projects( categoryId,pageSize,pageNo,search,sortedData ) {
+    static projects( categoryId,pageSize,pageNo,search,sortedData,language='en',hasSpotlight=false) {
         return new Promise(async (resolve, reject) => {
             try {
 
+                const defaultLanguage = 'en'
```

Comment:
@borkarsaish65 why we require defaultLanguage when you set language='en'?

Reply:
it will be helpful for search functionality. I have removed the below logic though.